core/modules/loader/CentralClient.cc

```diff
@@ -64,45 +65,45 @@ void CentralClient::start() {
 }
 
 
-void CentralClient::handleKeyInfo(LoaderMsg const& inMsg, BufferUdp::Ptr const& data) {
-    LOGS(_log, LOG_LVL_DEBUG, "CentralClient::handleKeyInfo");
+void CentralClient::handleKeyLook(LoaderMsg const& inMsg, BufferUdp::Ptr const& data) {
+    LOGS(_log, LOG_LVL_DEBUG, "CentralClient::handleKeyLook");
 
     StringElement::Ptr sData = std::dynamic_pointer_cast<StringElement>(MsgElement::retrieve(*data));
     if (sData == nullptr) {
-        LOGS(_log, LOG_LVL_WARN, "CentralClient::handleKeyInsertComplete Failed to parse list");
+        LOGS(_log, LOG_LVL_WARN, "CentralClient::handleKeyLook Failed to parse list");
         return;
     }
     auto protoData = sData->protoParse<proto::KeyInfo>();
     if (protoData == nullptr) {
-        LOGS(_log, LOG_LVL_WARN, "CentralClient::handleKeyInsertComplete Failed to parse list");
+        LOGS(_log, LOG_LVL_WARN, "CentralClient::handleKeyLook Failed to parse list");
         return;
     }
 
     // TODO put in separate thread
-    _handleKeyInfo(inMsg, protoData);
+    _handleKeyLook(inMsg, protoData);
 }
 
 
-void CentralClient::_handleKeyInfo(LoaderMsg const& inMsg, std::unique_ptr<proto::KeyInfo>& protoBuf) {
+void CentralClient::_handleKeyLook(LoaderMsg const& inMsg, std::unique_ptr<proto::KeyInfo>& protoBuf) {
```

Comment:
The intention is to put _handleKeyInfo in a separate thread at some point so that it doesn't use one of the boost asio threads. In which case it needs the pointer to keep the memory from being reclaimed. It may be just as well to just create more asio threads and pass a reference.